scripts/deploy.sh

```diff
@@ -50,14 +50,23 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   fi
   # force the deployment of haproxy for an AIO
   export DEPLOY_HAPROXY="yes"
-  if [[ ! -d /etc/openstack_deploy/ ]]; then
+  if [[ ! -e /openstack ]]; then
```

Comment:
The commit message mentioned that this would change to `/etc/openstack_deploy/openstack_user_config.yaml` not `/openstack`.  Unless I'm missing something, what gives?